src/index.ts

```diff
@@ -73,6 +73,17 @@ if (SIGNATURE_TYPE === undefined) {
   process.exit(1);
 }
 
+// CLI Help Flag
+if (process.argv[2] === '-h' || process.argv[2] === '--help') {
+  console.error(
+    `Example usage:
+  functions-framework --target=helloWorld --port=8080
+Documentation:
+  https://github.com/GoogleCloudPlatform/functions-framework-nodejs`
+  );
+  process.exit(1);
```

Comment:
I would expect a help command to exit on success,  errors reserved for outputting on another invalid command.

Reply:
Fixed.